### PR TITLE
fix typos in test and other files

### PR DESCRIPTION
**Pull request by allencloud**
fix typos in test and other files
Signed-off-by: Sun Hongliang <allen.sun@daocloud.io>

### CODE REVIEW

File: test/integration/api/create.bats [hunk 1]
```diff
@@ -11,14 +11,14 @@ function teardown() {
 	start_docker_with_busybox 2
 	swarm_manage
 
-	# make sure no contaienr exist
+	# make sure no contaienr exists
 	run docker_swarm ps -qa
```
Comment (thaJeztah) [inline test/integration/api/create.bats]: s/contaienr/container/ as well

**Comment by thaJeztah** [comment]
saw one more typo, but otherwise LGTM (not a maintainer in this repo)

**Comment by nishanttotla** [comment]
LGTM after the typo that @thaJeztah pointed out.

**Comment by allencloud** [comment]
Thanks @thaJeztah 
I have updated the PR.
@nishanttotla

**Comment by nishanttotla** [comment]
LGTM

**Pull request merged**